lib/beaker-hostgenerator/data.rb

```diff
@@ -329,48 +251,6 @@ def osinfo
             ],
           },
         },
-        # Deprecated
-        'cisconx-64' => {
-          :general => {
-            'platform'           => 'cisco_nexus-7-x86_64',
-            'packaging_platform' => 'cisco-wrlinux-5-x86_64',
-            'vrf' => 'management',
-            'ssh' => {
-              'user' => 'beaker',
-            },
-          },
-          :vmpooler => {
-            'template' => 'cisco-nxos-9k-x86_64',
-          },
-        },
-        # Deprecated
-        'ciscon7k-64' => {
-          :general => {
-            'platform'           => 'cisco_nexus-7k-x86_64',
-            'packaging_platform' => 'cisco-wrlinux-5-x86_64',
-            'vrf' => 'management',
-            'ssh' => {
-              'user' => 'admin',
-            },
-          },
-          :abs => {
-            'template' => 'cisco-n7k-7k-x86_64',
-          },
-        },
-        # Deprecated
-        'cisconxhw-64' => {
-          :general => {
-            'platform'           => 'cisco_nexus-7-x86_64',
-            'packaging_platform' => 'cisco-wrlinux-5-x86_64',
-            'vrf' => 'management',
-            'ssh' => {
-              'user' => 'devops',
-            },
-          },
-          :abs => {
-            'template' => 'cisco-nxos_hw-9k-x86_64',
-          },
-        },
         'cisco_n9k-VM' => {
```

Comment:
How much of cisco is still supported by Puppet?

Reply:
Is there even any cisco still supported? I don't think so

Reply:
Me neither, but perhaps someone can weigh in on this.

Reply:
We no longer support cisco 👍